posts/2023-07-20_cross_company_dev/cross_industry_dev.qmd

```diff
@@ -0,0 +1,148 @@
+<!-- You can read about the inception of [{admiral}](https://github.com/pharmaverse/admiral) in [this recent blog post by ...](). -->
+
+
+
+## Insights
+
+When creating an open source package in an industry where currently there is a lot of traction you have to move fast. 
+No, I don't mean: Move fast and break things. 
+I mean: Communicate! Get people on board! 
+We aimed [{admiral}](https://github.com/pharmaverse/admiral) to be **the** package for ADaM creation even before deciding to create the package together with GSK. 
+By being transparent about our endeavor, e.g. Thomas Neitmann (then at Roche) posting on LinkedIn, we managed to connect with Micheal Rimler from GSK and soon realized that we were dealing with the exact same challenge at both companies, and that a collaborative effort would improve the final product while reducing individual efforts.
+A working prototype of [{admiral}](https://github.com/pharmaverse/admiral) was to be created by GSK and Roche within six months, and would then be open sourced. 
+In our effort to communicate openly, we informed statistical programmers from over 20 companies about the [{admiral}](https://github.com/pharmaverse/admiral) project and invited them to try it out and provide feedback once released. 
+In the end, we received over 500 comments from over 50 programmers. 
+
+```{r}
+#| eval: false
+# check out the latest admiral release from CRAN:
+install.packages("admiral")
+
+library(admiral)
+```
+
+This was instrumental in creating a product that was optimised for general usage in the clinical reporting field and ensured that other companies would not unknowingly invest into their own solution to this challenge. 
+Open sourcing early is particularly beneficial because this ensures from the get-go that code created is aimed at a general audience, and not company specific (perhaps by accident).
+
+
+[{admiral}](https://github.com/pharmaverse/admiral) was created with the long term goal of having a stable and flexible solution for the clinical reporting pipeline. 
+In that spirit, its permissive [apache 2.0 licence](https://www.apache.org/licenses/LICENSE-2.0) (jointly owned by Roche and GSK) further strengthens trust into the project, namely for three reasons: 
+
+  - Jointly owned means that efforts to monetize the code-base by one company could be vetoed by the other. The permissive licence ensures that in such an unlikely case, the code-base that has been published would stay available and could always be used and improved upon by others. 
+
+  - Having this package backed by Roche and GSK ensures (as far as this is ever possible) funding for properly maintaining the packages. This is crucial, as a package is rarely **finished**. 
+
+Up to now, experts from many more companies have joined [{admiral}](https://github.com/pharmaverse/admiral) or one of its therapeutic area specific package-extensions, inspiring even more trust into its reliability.
```

Comment:
@StefanThoma  i think this should be bullet pointed as seems like the 3rd reason in your list